runtimes/azure-client-runtime/src/main/java/com/microsoft/azure/AzureEnvironment.java

```diff
@@ -83,9 +83,9 @@ public AzureEnvironment(
             "https://management.usgovcloudapi.net/");
 
     /**
-     * Provides the settings for authentication with Azure German Government.
+     * Provides the settings for authentication with Azure German.
      */
-    public static final AzureEnvironment AZURE_GERMAN_GOVERNMENT = new AzureEnvironment(
+    public static final AzureEnvironment AZURE_GERMAN = new AzureEnvironment(
```

Comment:
It's all named randomly as long as it's intuitive. 
```bash
$ azure account env list
info:    Executing command account env list
data:    Name
data:    -----------------
data:    AzureCloud
data:    AzureChinaCloud
data:    AzureUSGovernment
data:    AzureGermanCloud
info:    account env list command OK
```
this is the names in Xplat CLI. But I would vote for AZURE_GERMANY to be consistent with AZURE_CHINA.